### DISCUSSION TITLE
connection error

### DISCUSSION THREAD

**Discussion by simple0313** [projectdiscovery/interactsh#586] [Q&A]
Server uses CentOS 7,
client uses Mac M1,
encountering connection error.
![image](https://github.com/projectdiscovery/interactsh/assets/31197600/e90d1424-8276-4656-aab5-bf4758c192ba)

**Comment by ehsandeep** [comment]
@simple0313 looks like you are using self-hosted server, make sure you are using the latest version + correctly hosted your self-hosted version of server.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
